Create zigbee.markdown as start of Zigbee documentation

Hedda opened the pull request:
**Description:**

Creating zigbee.markdown as a start of a ZHA documentation even if it is almost only a placeholder.

The base of this Zigbee text was mainly copied from the Z-Wave documentation for Home Assistant.

My purpose of this is to at least get some information about the existing Zigbee Home Automation integration into the main documentation as I think that this could be used as a base and a start for other technical writers who can build on this and help improve the Zigbee documentation for Home Assistant.

With this, I also wish to expose the existing Zigbee Home Assistant integration and show it in the same light as the existing Z-Wave integration. I hope that this will help the existing  Zigbee Home Assistant integration become more of a first-class citizen in Home Assistant, just like the existing Z-Wave integration already is.

Ping @dmulcahey @Adminiuga & @damarco for feedback request on this documentation content.

PS: This PR is for the 'current' branch and replaces #10582 which was based on the 'next' branch.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Hedda committed:
Create zigbee.markdown as start of Zigbee documentation (current branch)

Creating zigbee.markdown as a start of a ZHA documentation even if it is almost only a placeholder.

The base of this Zigbee text was mainly copied from the Z-Wave documentation for Home Assistant.

The purpose of this is just to get Zigbee into the main documentation that anyone can build on.

Ping @dmulcahey @Adminiuga & @damarco for feedback request on this documentation content.

PS: This PR is for the 'current' branch and replaces #10582 which was based on the 'next' branch.

frenck commented on source/_docs/zigbee.markdown:
```diff
@@ -0,0 +1,37 @@
+---
+title: "Zigbee"
+description: "Using Zigbee with Home Assistant."
+redirect_from: /getting-started/zigbee/
```
? Why this redirect?

Hedda replied:
Miss from when first copied the main text from the Z-Wave documentation. Removed now.

Hedda committed:
Add link for zigbee.markdown (starter Zigbee doc) to index.markdown

Adding a link for zigbee.markdown (starter Zigbee doc) to index.markdown (current branch). 

The purpose is to expose the Zigbee Home Assistant integration and show in the same light as the Z-Wave integration. I hope that this will help the existing Zigbee integration become more of a first-class citizen in Home Assistant, just like the existing Z-Wave integration already is.

frenck commented:
My initial thought with this is: We should not do this.

Reason for this is:
- Z-Wave docs are scattered around the site and that makes it hard for the user to find things.
- We have integration documentation for these things.
- ZHA is not the only Zigbee solution, which is ignored in this PR.

I think this is just a little too soon. We should definitely revisit parts of the website & documentation, which should start at revisiting our general documentation and website structure.

Hedda committed:
Remove redirect

Redirect was miss when copied from Z-Wave documentation

Hedda commented:
@frenck the last section covers some (non-native) alternative ways of controlling Zigbee devices from HA. 

_"Other than using the above described native [Zigbee Home Automation](/integrations/zha/) integration in Home Assistant, there are also other integrations for Home Assistant connect a third-party bridge, hub or gateway (that can be hardware or software) which in turn can also communicate with Zigbee devices, through those other integrations mostley or fully absract the fact that the devices are using the Zigbee specification (as such that the devices could just as well be using any other specification). Such integrations with third-party bridges, hubs and gateways that can also communicate with Zigbee devices include but are not exclusive to; deCONZ, Zigbee2mqtt, Philips Hue Hub, Ikea Tradfri Gateway, Osram Lightify, and Samsung SmartThings Hub."_

You could directly compare this to the fact that it is as far as I could see no mentions in the Z-Wave documentation that controlling Z-Wave devices is also possible through non-native integration with third-party bridges, hubs and gateways. Therefore this Zigbee documentation is possibly more fair by at least listing other options.

frenck commented:
You are missing my point. The point is that the current structure (including Z-Wave) is not good IMHO. That is why I don't feel like we should extend that in that way.

This has nothing to do with any class of citizenship.

fabaff commented:
I'm with @frenck on this. We should review the structure first.

A while back we decided to move Z-Wave and MQTT away from the integration documentation to the "Docs" section. The reason was that both integration provided a huge amount of details which made the integration documentation a pain to read. Also, it was confusing for users to see that much information which was not relevant for their use case.

frenck commented:
Thanks for providing the second opinion on this matter, fabaff.

So I'm going to close this, for now, let's revisit this in the future. Nevertheless, @Hedda, thank you for your contribution! That is appreciated.
